scripts/insert_into_appointments.sql

```diff
@@ -1 +1,10 @@
 -- insert appointments creating connections between doctors and patients
+
+-   Marsha Wilcox seeing Dr. Vicky Hilton at 10am on August 12, 2016.
+-   Ivory Talley seeing Dr. Spencer Hubbard at 1pm on August 17, 2016.
+-   Marsha Wilcox seeing Dr. Spencer Hubbard at 10am on August 12, 2016.
+-   Blake Hodge seeing Dr. Vicky Hilton at 1pm on August 17, 2016.
+INSERT INTO appointments(date, time, doctor_id, patients_id, notes)
+SELECT '2016-08-12', '1pm', d.id, p.id
+FROM doctor d, patients p
+  WH
```

Comment:
Looks like you ran out of time, but you were totally on the right track. Your solution would of course need to match the way your table was set up, but here's 1 way you could write this script:

```sql
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT '20150812', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hilton'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Talley')
  UNION
  SELECT '20150817', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
   (SELECT id FROM doctors WHERE surname = 'Hilton'),
   (SELECT id FROM patients WHERE surname = 'Hodge')
;
```